### ISSUE TITLE
Pre-trained VAEGAN model uses older batch normalization implementation

### ISSUE THREAD

**Issue by pkmital** [pkmital/CADL#62]
https://www.kadenze.com/forums/file-submission-generative-adversarial-networks-and-recurrent-neural-networks/threads/anyone-else-seen-this-issue-before

Should retrain a new model on the newer batch norm implementation in tf.contrib
